third_party/repositories.bzl

```diff
@@ -64,7 +64,7 @@ swift_c_module(
         module_map = module_map,
     ))
 
-def namespaced_swift_library(name, srcs, deps = None, cc_libs = None, defines = None):
+def namespaced_swift_library(name, srcs, deps = None, cc_libs = None, defines = None, copts=[]):
```

Comment:
We can just combine `cc_libs` into `deps` since this is all handled internally in the rules_swift rules

Reply:
Seems good